<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1">
    <widget class="QGroupBox" name="parametersBox">
     <property name="title">
      <string>Paramètre</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QComboBox" name="baudRateBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="parityBox"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="stopBitsBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="stopBitsLabel">
        <property name="text">
         <string>Bits de stop :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="flowControlLabel">
        <property name="text">
         <string>Contrôle de flux :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="dataBitsLabel">
        <property name="text">
         <string>Nombre de bits de Données :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="flowControlBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="dataBitsBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="parityLabel">
        <property name="text">
         <string>Parité :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="baudRateLabel">
        <property name="text">
         <string>Vitesse :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="serialPortInfoListBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>96</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_Appliquer">
       <property name="text">
        <string>Appliquer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
